lib/logstash/outputs/opsgenie.rb

```diff
@@ -6,6 +6,27 @@
 require "net/http"
 require "net/https"
 
+
+HTTP_EXCEPTIONS = [
```

Comment:
I tried to select those looking into their meaning in ruby docs and other online sources but I am happy to deliberate if they should be retried as well. In general they are in group of request badly formed or response impossible so I do not think we should be retrying them.